PICMI_Python/diagnostics.py

```diff
@@ -329,3 +329,48 @@ def __init__(self, grid, num_snapshots, dt_snapshots, data_list=None,
         self.name = name
 
         self.handle_init(kw)
+
+class PICMI_ParticleBoundaryScrapingDiagnostic(_ClassWithInit) :
```

Comment:
A better place for this class would be right after `PICMI_ParticleDiagnostic` (before the `Lab frame diagnostics` diagnostics comment).